lib/feed/deleteFeedItem.ts

```diff
@@ -0,0 +1,14 @@
+import { doc, deleteDoc } from "firebase/firestore";
+
+import { getCommentsCol, feedCol } from "../firestore";
+
+const deleteFeedItem = async (postId: string, commentId?: string) => {
+  if (commentId) {
+    await deleteDoc(doc(getCommentsCol(postId), commentId));
+  } else {
+    await deleteDoc(doc(getCommentsCol(postId)));
```

Comment:
Actually, this isn't recommended by firestore (and doesn't work), so just take out this line. Sorry, I didn't realize.
```suggestion
```